app/create-profile/page.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { getServerSession } from 'next-auth'
+import { authOptions } from '@/lib/auth'
+import { redirect } from 'next/navigation'
+
+const CreateProfilePage = async () => {
+  const session = await getServerSession(authOptions)
+
```

Comment:
**issue:** React component should always return something. In case of you don't want to return anything you have to return `null`. However in that case let's do just:

```tsx
const CreateProfilePage = async () => {
  const session = await getServerSession(authOptions)

  if (!session) {
    redirect('/')
  }
  
  return <div>Create profile page</div>
}
```